test/Microsoft.IdentityModel.Tokens.Extensions.Tests/KeyVaultKeyWrapProviderTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.KeyVault;
+using Microsoft.IdentityModel.Tests;
+using Xunit;
+
+#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
+
+namespace Microsoft.IdentityModel.Tokens.Extensions.Tests
+{
+    public class KeyVaultKeyWrapProviderTests
+    {
+        private readonly IKeyVaultClient _client;
+        private readonly SecurityKey _key;
+
+        public KeyVaultKeyWrapProviderTests()
+        {
+            _client = new MockKeyVaultClient();
+            _key = new KeyVaultSecurityKey(KeyVaultUtilities.CreateKeyIdentifier(), keySize: default);
+        }
+
+        public static TheoryData<KeyWrapProviderTheoryData> DisposeProviderTheoryData
+        {
+            get => new TheoryData<KeyWrapProviderTheoryData>
+            {
+                new KeyWrapProviderTheoryData
+                {
```

Comment:
Maybe put "First = true" here just to be consistent?

Reply:
Fixed: Added "First = true"